src/main/java/sc/fiji/pQCT/selectroi/RoiSelector.java

```diff
@@ -385,7 +385,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 		return returnVectorVectorPointer;
 	}
 
-	private byte[] dilate(final byte[] data, final byte dilateVal, final byte min,
+	protected byte[] dilate(final byte[] data, final byte dilateVal, final byte min,
 		final byte temp)
```

Comment:
Why `protected`?

Reply:
I could not call the private method from a class that extended this class.

Reply:
Fair enough. I would be cautious with using `protected` visibility. It's hard to keep track of shared fields and methods between super and sub classes.